lua/outline/config.lua

```diff
@@ -90,7 +90,7 @@ M.defaults = {
     up_and_jump = '<C-k>',
   },
   providers = {
-    priority = { 'lsp', 'coc', 'markdown', 'norg' },
+    priority = { 'lsp', 'coc', 'markdown', 'norg', 'vimdoc'},
```

Comment:
Remember to reflect changes to the default config in the relevant section of readme too.